portfolio/src/main/webapp/script.js

```diff
@@ -27,40 +27,77 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
-let turn;
-//initializes the Tic-Tac-Toe Grid
+
+//initializes the Tic-Tac-Toe Grid, triggered by both the play and reset buttons
 function initTic(){
-  turn = 'X';
-  var tic = document.getElementById("tic");
-  var paper = document.createElement("div");
+  let turn = 'X';
+  let tic  = document.getElementById("tic");
+  let paper = document.createElement("div");
   paper.classList.add("paper");
   tic.appendChild(paper);
-  var boxGrid = Array(3).fill(Array(3).fill(""));
-  var checkGrid = Array(3).fill(Array(3).fill(""));
-  for(var i=0;i<3;i++){
+  let boxGrid =[];
```

Comment:
Whitespace is a bit inconsistent throughout this file. The style guide requires putting whitespace around operators.
https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace
Horizontal whitespace "On both sides of any binary or ternary operator."